source/whats-new.txt

```diff
@@ -82,6 +82,77 @@ The 3.0 driver release includes the following new features:
   `TimeOnly Struct. <https://learn.microsoft.com/en-us/dotnet/api/system.timeonly?view=net-6.0>`__
   API reference page on MSDN.
 
+- Adds support for implicit client-side projection when using the ``Find()`` method,
+  the ``Select()`` method, or the ``Project()`` aggregation stage with the LINQ3 provider.
+  In previous versions of the driver, you could perform client-side
+  projection with the LINQ3 provider only after calling the ``ToEnumerable()`` or
+  ``AsEnumerable()`` method, and only within the ``Select()`` method.
```

Comment:
> and only within the ``Select()`` method

Why would this be the case? Once you're client-side you can do anything.

Reply:
I suspect I inferred this from something in the Jira ticket.